core/src/main/java/org/opensearch/sql/ast/AbstractNodeVisitor.java

```diff
@@ -289,4 +290,8 @@ public T visitQuery(Query node, C context) {
   public T visitExplain(Explain node, C context) {
     return visitStatement(node, context);
   }
+
+  public T visitNestedFunction(NestedFunction node, C context) {
+    return visitChildren(node, context);
+  }
```

Comment:
So nested function is special node just like highlight function. Any chance to make it just a regular function registered to function repository? How can we move this to `opensearch` module? As discussed, all OpenSearch specific functions including relevancy search and highlight should be moved out of `core` soon.

Reply:
Has been updated as a regular function. I am not sure which approach will be needed for moving necessary functions to the `opensearch` module.